packages/components/psammead-consent-banner/src/index.stories.jsx

```diff
@@ -66,3 +70,5 @@ storiesOf('Components|ConsentBanner', module)
     },
     { notes, knobs: { escapeHTML: false } },
   );
+
+buildRTLSubstories(STORY_KIND, { include: ['default'] });
```

Comment:
The consent banner example has three levels of nesting for a single example. They all seem to show the same Arabic example, which looks good! Would be good to understand why it's repeating like this.
<img width="214" alt="Screenshot 2019-10-23 at 17 36 41" src="https://user-images.githubusercontent.com/3028997/67414835-be2fbd80-f5bb-11e9-820c-728b5795c096.png">

Reply:
I ran `npm run ci:packages` again and it no longer has this duplication. So I'm resolving this comment.